fightScreen.py

```diff
@@ -0,0 +1,633 @@
+import sys
+from random import choice as randChoice
+from time import sleep
+from PySide6.QtCore import QSize, Qt, QTimer
+from PySide6.QtGui import QPixmap, QMovie, QColor
+from PySide6.QtWidgets import (
+    QApplication,
+    QMainWindow,
+    QLabel,
+    QWidget,
+    QGridLayout,
+    QSpacerItem,
+    QSizePolicy,
+    QPushButton,
+)
+from meter import Meter
+from arsenal import Arsenal
+
+
+class MainWindow(QMainWindow):
+    def __init__(self):
+        super().__init__()
+
+        self.imageDir = "C:/classGame/"
+
+        self.userActionArray = []
+        self.compActionArray = []
+        self.imgDict = {
+            "sword": QPixmap(self.imageDir + "sword.jpg"),
+            "spear": QPixmap(self.imageDir + "spear.jpg"),
+            "shield": QPixmap(self.imageDir + "shield.jpg"),
+            "gloves": QPixmap(self.imageDir + "gloves.jpg"),
+        }
+        punch = QMovie(self.imageDir + "P1Attack1.gif")
+        kick = QMovie(self.imageDir + "P1Attack2.gif")
+        defense = QMovie(self.imageDir + "P1Defense.gif")
+        self.actionArray = ["Punch", "Kick", "Defend"]
+        self.damageArray = [10, 20, 0]
+        self.player1PicArray = [punch, kick, defense]
+        self.P1WeaponArray = {"sword": [20, 30, 10], "spear": [30, 10, 20]}
+        self.P1DefenseArray = {"shield": [30, 10, 20], "gloves": [10, 10, 10]}
+        self.P2WeaponArray = {"sword": [20, 30, 10], "spear": [30, 10, 20]}
+        self.P2DefenseArray = {"shield": [30, 10, 20], "gloves": [10, 10, 10]}
+        self.P1HealthMeter = 100
+        self.P2HealthMeter = 100
+        self.P1MagicMeter = 100
+        self.P2MagicMeter = 100
+        self.P1MechMeter = 100
+        self.P2MechMeter = 100
```

Comment:
When @NoahBartell completes #18 we should be able to get rid of all this.

Reply:
Let's have this as an issue to assign after merge.

Reply:
Oh yeah good idea.